### ISSUE TITLE
Code inside Mixin displayed as "unreachable"

### ISSUE THREAD

**Issue by Infinituum17** [DemonWav/MinecraftDevIntelliJ#2323]
### Minecraft Development for IntelliJ plugin version

2024.1-1.7.6

### IntelliJ version

2024.1.1

### Operating System

Windows 11

### Target platform

Fabric, Mixins

### Description of the bug

I'm not 100% sure if it's an issue with this extension but the code I've written inside my mixin class is displayed as "unreachable" (grey text).

NOTE: The mixin works fine but I didn't know if I had to report this bug here or somewhere else

The code:
```java
@Mixin(Slot.class)
public class SlotMixin {
    @Shadow
    @Final
    public Container container;

    @ModifyVariable(method = "set", at = @At("HEAD"), argsOnly = true)
    public ItemStack set(ItemStack itemStack) {
        ExceptionalStatsObject stack = ((ExceptionalStatsObject) (Object) itemStack);

        if (stack == null) return itemStack;

        if (container instanceof Inventory && stack.exceptionalstats$shouldHaveStatModifiers()) {
            stack.exceptionalstats$addAllStatModifiers();
        }

        return itemStack;
    }
}
```

![image](https://github.com/minecraft-dev/MinecraftDev/assets/40234342/06cdface-5731-482e-9d9c-f994d9a394ad)

**Comment by RedNesto** [comment]
Looks like a duplicate of #2284

**Comment by Infinituum17** [comment]
I didn't check but there was a new version of Intellij available (2024.1.4), so I updated it and now it displays everything correctly.

Sorry if I wasted your time
